### ISSUE TITLE
Some questions about training 

### ISSUE THREAD

**Issue by yuyijie1995** [experiencor/keras-yolo3#26]
![2018-04-22 21-15-17](https://user-images.githubusercontent.com/30818917/39095499-186719e0-4674-11e8-8348-135a7cec6cf1.png)
1.What is the [13],[26],[52]'s meaning? 
2.Why the loss is so high , different from the normal loss like 0.355?

**Comment by experiencor** [comment]
1. It's the size of the grid.
2. It starts out that big but will get reduced after each batch.

**Comment by chintan5750** [comment]
My training is getting stopped after 1st epoch in Ubuntu. Have you hear this kind of issue ?

**Comment by FMsunyh** [comment]
@experiencor  I don't want print this log, how can I do?
![image](https://user-images.githubusercontent.com/9877644/39855472-a45e2388-545e-11e8-9c2b-521320555e6e.png)
